### PR TITLE
Feat/paas migration produce test set

### PR DESCRIPTION
**Pull request by JamesPRobinson**
See https://uktrade.atlassian.net/browse/DT-2347?atlOrigin=eyJpIjoiYmRmZjA3NTA5NjU3NDNhNGJlYTFkM2FiNWM0MjYzZmQiLCJwIjoiaiJ9

Also a change to env token formatting for CSRF_TRUSTED_ORIGINS required from Django update, can no longer specify '*'

### CODE REVIEW

File: docs/test_suite.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Test suite
+
+Documents manual testing against the new platform to ensure all business as usual activities can be performed.
+
+## Helpcentre CMS Platform
+
+### 1. Adding articles to HelpCentre
+- Access the admin area of the CMS (Content Management Service) at `/admin`.
+- Click the icon 'Page', this sould take you to the page 'Home'.
```
Comment (rav-pradhan) [inline docs/test_suite.md:9]:
```suggestion
- Click the icon 'Page', this should take you to the page 'Home'.
```

File: docs/test_suite.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+- CLick the plus icon at the top of the page (next to the title 'Home'). It should have the tooltip 'Add child page'.
```
Comment (rav-pradhan) [inline docs/test_suite.md:10]:
```suggestion
- Click the plus icon at the top of the page (next to the title, 'Home'). It should have the tooltip, 'Add child page'.
```

File: docs/test_suite.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Click 'Article index page', fill out the fields and then click 'Publish' below (default is 'Save draft').
+![alt text](img/test_index.png "Test Index Article")
+
+_Go to the home page and..._
+
+**Check all fields entered in CMS are rendered** [x]
```
Comment (rav-pradhan) [inline docs/test_suite.md:16]: Are the `[x]` throughout checklist items to confirm if the process is correct?

File: docs/test_suite.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+- There should be a 'Play' button in the right-hand corner to run the DAG, click this to manually run the DAG and consume the feedback data from the helpcentre.
+- You should see a 'Success' run with a dark green bar in the left-hand side chart like the image below:
+
+_Access data-flow having submitted feedback_
+
+![alt text](img/test_data-flow.png "Test data-flow feedback accepted by pipeline")
+
+**Check that Data-flow can talk to helpcentre and that the DAG run is successful having submitted feedback** [x]
+
+### 6. DataHub can access articles via API
```
Comment (rav-pradhan) [inline docs/test_suite.md:70]: Does Data Hub access HelpCentre articles through a public API? I'm wondering whether it's feasible to test this in this way prior to cutover. If not possible, then I suppose we should be able to check the endpoint is accessible on our side, and returns the correct response?

**Review by rav-pradhan — COMMENTED** [review]
I can't comment on whether all necessary activities have been covered here, but the instructions of what has been included were clear to me. I've left a few spelling, punctuation, grammar suggestions, and also a comment on testing the Data Hub feed access

File: docs/test_suite.md (hunk 3 above)
Comment (JamesPRobinson) [reply docs/test_suite.md:16]: They're there to signal that this is a checklist task, I can remove if confusing

File: docs/test_suite.md (hunk 4 above)
Comment (JamesPRobinson) [reply docs/test_suite.md:70]: If you're happy to do that, I think that's a better way of doing it. Same goes for the DataHub articles/ endpoint. Let me know and I'll change the instructions here.

File: docs/test_suite.md (hunk 3 above)
Comment (rav-pradhan) [reply docs/test_suite.md:16]: Nah that's all good, cheers for the context

File: docs/test_suite.md (hunk 4 above)
Comment (rav-pradhan) [reply docs/test_suite.md:70]: I've asked the rest of the migration team about this. Hopefully can get back to you soon

Comment (JamesPRobinson) [reply docs/test_suite.md:70]: Thanks for checking Ravi, I'll speak to them and then update the markdown

Comment (JamesPRobinson) [reply docs/test_suite.md:70]: Have checked in with DataHub team, they're happy for us to update DataHub staging

File: docs/test_suite.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+![alt text](img/test_article_amended.png "Test Index Article Content Amended")
+
+**Check all fields entered in CMS are rendered** [x]
+
+**Check this 'Was this page helpful?' widget renders** [x]
+
+### 3. Checking article pages cannot be created from the Homepage
+ - Access the admin area.
+
+### 3. Deleting articles on HelpCentre
```
Comment (ian-leggett) [inline docs/test_suite.md:50]: Should this be ### 4?

**Review by ian-leggett — APPROVED** [review]

File: docs/test_suite.md (hunk 5 above)
Comment (JamesPRobinson) [reply docs/test_suite.md]: good catch!

**Pull request merged**